### PR TITLE
Pg later

### PR DESCRIPTION
**Pull request by ChuckHend**

### CODE REVIEW

File: pg_later/src/bgw.rs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+use pgrx::bgworkers::*;
+
+use crate::api::{delete_from_queue, get_job, query_to_json};
+
+#[pg_guard]
+pub extern "C" fn _PG_init() {
+    BackgroundWorkerBuilder::new("PG Later Background Worker")
+        .set_function("background_worker_main")
+        .set_library("pg_later")
+        // .set_argument(42i32.into_datum())
```
Comment (ianstanton) [inline pg_later/src/bgw.rs:16]:
```suggestion
```

File: pg_later/src/bgw.rs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        .enable_spi_access()
+        .load();
+}
+
+#[pg_guard]
+#[no_mangle]
+pub extern "C" fn background_worker_main(_arg: pg_sys::Datum) {
+    // let arg = unsafe { i32::from_polymorphic_datum(arg, false, pg_sys::INT4OID) };
```
Comment (ianstanton) [inline pg_later/src/bgw.rs:24]:
```suggestion
```

File: pg_later/src/bgw.rs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                None => {
+                    log!("No job");
+                }
+            }
+            Ok(())
+        });
+    }
+
+    log!(
+        "Goodbye from inside the {} BGWorker! ",
```
Comment (ianstanton) [inline pg_later/src/bgw.rs:58]: :wave:

File: pg_later/src/api.rs [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+}
+
+// #[pg_extern]
+// pub fn exec_to_table(
+//     query: &str,
+// ) -> Result<TableIterator<'static, (name!(query, String), name!(results, pgrx::JsonB))>, spi::Error>
+// {
+//     let resultset = query_to_json(query)?;
+//     Ok(TableIterator::new(resultset.into_iter()))
+// }
```
Comment (ianstanton) [inline pg_later/src/api.rs:105]:
```suggestion
```

**Review by ianstanton — APPROVED** [review]
Looks good, just some cleanup suggestions!

File: pg_later/src/api.rs (hunk 4 above)
Comment (ChuckHend) [reply pg_later/src/api.rs:105]: I'm going to keep this one for now since its still in development.

**Pull request merged**